app/controllers/items_controller.rb

```diff
@@ -5,6 +5,7 @@ def browse
   end
 
   def index
+    @extra_js = [javascript_include_tag("search")]
```

Comment:
Sounds good to me! I just copied how we did it in L&C so we should probably change it there too. Would you mind updating it over there and I'll fix it in orchid?